robot_smach_states/src/robot_smach_states/manipulation/grab.py

```diff
@@ -129,7 +134,7 @@ def execute(self, userdata):
         else:
             rospy.loginfo("Updated pose of entity (dx, dy, dz) : (%f, %f, %f)" %
                           (updated_grab_entity.pose.position.x - grab_entity.pose.position.x,
```

Comment:
Grab entities are still geometry msgs. Otherwise you could, e.g., loginfo updated_grab_entity.pose.p - grab_entity.pose.p